### ISSUE TITLE
Feature request: allow to configure sauce connect version

### ISSUE THREAD

**Issue by abendt** [aginteractive/Gradle-Sauce-Connect-Plugin#13]
hi, 

This is a feature request.

we are currently experiencing stability issues  using the sauce connect tunnel. after a conversation with the sauce labs support it seems that the problem is at their end with the latest version of sauce connect (v4.5.3). Their current recommendation is to use v4.5.1 until the issue is resolved.

However it seems that the gradle-sauce-connect-plugin always uses the latest version of sauce connect. I think in general it would be helpful, if the plugin allowed to also configure a specific version of sauce connect to use.

**Comment by johnroach** [comment]
Yeah, as an explanation we use SauceLab's endpoint to inform us of available versions which is https://saucelabs.com/versions.json We also use the available checksum to avoid corrupted downloads.

A good solution would be to allow developers to provide URL and digest as an option. Both which could be found from https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy

**Comment by abendt** [comment]
@johnroach yes, i saw that in the sourcecode of the plugin. actually its really conveninent that the plugin determines the latest version automatically.

based on your comment and our needs i experimented a bit and came up with this version: https://github.com/abendt/Gradle-Sauce-Connect-Plugin

if it is acceptable for you i could make a PR out of it.
